sdk/nodejs/image.ts

```diff
@@ -121,7 +128,7 @@ export class Image extends pulumi.CustomResource {
      */
     public /*out*/ readonly registryServer!: pulumi.Output<string>;
     /**
-     * The digest of the manifest pushed to the registry, e.g.: repo[:tag]@<algorithm>:<hash>
+     * The digest of the manifest pushed to the registry, e.g.: repository@<algorithm>:<hash>
      */
     public /*out*/ readonly repoDigest!: pulumi.Output<string | undefined>;
```

Comment:
It could be handled separately - but as called out in the feedback motivating this docs change - it’s *really* unfortunate that this is an optional output. Makes it exceedingly hard to use in practice. 

I really do think we should consider making it always available - and just defaulting to the image name if not pushed to a repo. That would allow it to be used without condition as the “most reliable unique name for the resulting image available”. Which is what people want.

Reply:
Would it be possible to decide this in a follow-up issue? This PR was meant as a first pass at docs clarification.

I agree that making this field Required would make it much easier to use.

However, having a field called `repoDigest` whose value doesn't include a digest (in the case of a local image) doesn't sit right with me. 

I think I'd prefer a scenario where "repoDigest" means "manifest SHA" for pushed images, and something like "the docker inspect image ID field" for local (non-pushed) images.